tests/unit/src/main/scala/scala/meta/internal/metals/debug/TestDebugger.scala

```diff
@@ -103,7 +103,11 @@ final class TestDebugger(
   def shutdown: Future[Unit] = {
     Debug.printEnclosing()
     for {
-      _ <- terminated.future
+      _ <- terminated.future.withTimeout(60, TimeUnit.SECONDS).recoverWith {
+        case _ =>
+          scribe.warn("We never got the terminate message")
```

Comment:
If the test passes anyway, then the sending terminate message might be flaky, but let's not fail the test on that.